content/en/real_user_monitoring/guide/monitor-electron-applications-using-browser-sdk.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Monitor Electron Applications Using the Browser SDK
+kind: guide
+beta: true
+description: Learn how to monitor desktop applications built with Electron through the browser RUM SDK.
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+<div class="alert alert-warning">
+    Electron support for RUM is in beta.
+</div>
+
+## Overview
+
+[Electron][1] is an open source framework that can be used to build cross-platform macOS and Windows desktop applications.
+
+You can install and configure the Datadog Browser SDK to start monitoring applications built using Electron JS.
+
+**Note**: The Browser SDK only supports monitoring the *renderer processes* of an application. It does not initialize or monitor anything installed on the *main process*. For more information, see Electron's documentation on its [renderer process][3].
+
+{{< img src="real_user_monitoring/guide/monitor-electron-applications-using-browser-sdk/diagram_electron-js-browser-rum.png" alt="The Datadog Browser SDK only supports monitoring the renderer processes of an Electron application" style="width:100%" >}}
+
+## Installation
+
+To install the Datadog Browser SDK to support Electron apps:
+
+1. Set up and install [RUM Browser Monitoring][2] inside **every renderer process**, following the steps for CDN sync, CDN async, or npm.
+
+2. Set the `allowFallbackToLocalStorage` parameter to true in the RUM initialization configuration of each renderer process, as shown below. 
+
+   **Note**: This setting allows Datadog to collect RUM without relying on browser cookies.
```

Comment:
This sentence reads a little oddly to me. Is there another word you can add after RUM? Perhaps "data," so the sentence would read something like "allows Datadog to collect RUM data without relying on..."